OWNERS

```diff
@@ -1,11 +1,23 @@
 # See the OWNERS docs at https://go.k8s.io/owners
+filters:
+  ".*":
+    reviewers:
```

Comment:
did you mean to change half of this group to reviewers instead of approvers?

Reply:
Well, as it was said to have you to review them that was the goal. Would you suggest to adapt it in other way?

Reply:
just the blogs / technical content - i don't want to be in the critical path for everything :)

Reply:
Tried to adjust it, let's see if it works as expected :-)

Reply:
I don't think it hurts anything, but I believe listing people as reviewers is redundant if they're already listed as approvers.  I don't think being a reviewer gives you anything more than what you get by default as a member of the github org with our setup.